tests/common/factories/group.py

```diff
@@ -20,7 +20,6 @@ class Meta:
     joinable_by = JoinableBy.authority
     readable_by = ReadableBy.members
     writeable_by = WriteableBy.members
-    members = factory.LazyFunction(list)
```

Comment:
This has to be removed because `Group.members` is now a read-only property. I'm not replacing this with a `memberships` attribute in the factory: this would be pointless because `models.Group.memberships` defaults to `[]` anyway. The `members` attribute here was never necessary in the first place